internal/pkg/plugin/gitlabcedocker/gitlabcedocker.go

```diff
@@ -0,0 +1,59 @@
+package gitlabcedocker
+
+import "strings"
+
+// TODO(dtm): Add your logic here.
+
+const (
+	gitlabImageName = "gitlab/gitlab-ce:rc"
```

Comment:
Perhaps the configuration of the image version can be opened up, but there is no rush, it can be left until the next pr